### PR TITLE
OADP-717: OADP 1.0.x & OCP 4.11 compatibility issue

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.1.1 and OADP 1.0.x; OCP 4.11+

Dev and QE review complete. OCP peer review complete and Avital's suggestion implemented. 

Resolves https://issues.redhat.com/browse/OADP-717 by adding a note to the sections shown in the previews.

Previews: 
1. See note in https://51355--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/index.html#oadp-requirements

2. See 2nd note in https://51355--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html

### CODE REVIEW

File: snippets/oadp-ocp-compat.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+//This snippet appears in the following assemblies:
+//
+// * .../backup_and_restore/backing_up_and_restoring/installing/about-installing-oadp.adoc
+// * .../backup_and_restore/index.adoc
+
+:_content-type: SNIPPET
+[NOTE]
+====
+OADP 1.0._x_ does not support CSI backup on OCP 4.11+. because Velero 1.7.x, which ships with OADP 1.0._x_, expects the API group `snapshot.storage.k8s.io/v1beta1`,` which is not present on OCP 4.11+.
```
Comment (rayfordj) [inline snippets/oadp-ocp-compat.adoc:10]: There's an extraneous back-tick after the comma that should be removed.

```suggestion
OADP 1.0._x_ does not support CSI backup on OCP 4.11+. OADP 1.0._x_ ships Velero 1.7.x and expects the API group `snapshot.storage.k8s.io/v1beta1`, which is not present on OCP 4.11+.
```

**Review by rayfordj — APPROVED** [review]
/lgtm

Thanks, @RichardHoch! :+1:

**Comment by RichardHoch** [comment]
oadp

**Comment by RichardHoch** [comment]
OADP

File: snippets/oadp-ocp-compat.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+//This snippet appears in the following assemblies:
+//
+// * .../backup_and_restore/backing_up_and_restoring/installing/about-installing-oadp.adoc
+// * .../backup_and_restore/index.adoc
+
+:_content-type: SNIPPET
+[NOTE]
+====
+OADP 1.0._x_ does not support CSI backup on OCP 4.11+. OADP 1.0._x_ includes Velero 1.7._x_ and expects the API group `snapshot.storage.k8s.io/v1beta1`, which is not present on OCP 4.11+.
```
Comment (apinnick) [inline snippets/oadp-ocp-compat.adoc:10]: Very small nit: "4.11+". Our style guides specify "later" or "later versions" rather than the "+".

**Comment by RichardHoch** [comment]
OADP 1.1.1 GA is 28 November.

**Comment by jboxman-rh** [comment]
@RichardHoch, should this not be merged until that time?

Thanks!

**Comment by RichardHoch** [comment]
@jboxman-rh -- That is my understanding; I will double-check but please don't merge yet. Thanks for asking!

**Comment by RichardHoch** [comment]
@jboxman-rh please merge this

**Comment by RichardHoch** [comment]
/OADP
/merge-review-needed
was approved for merging; today is the day

**Pull request merged**